package/MDAnalysis/core/groups.py

```diff
@@ -2531,7 +2531,7 @@ def improper(self):
         return topologyobjects.ImproperDihedral(self.ix, self.universe)
 
     def write(self, filename=None, file_format="PDB",
-              filenamefmt="{trjname}_{frame}", **kwargs):
+              filenamefmt="{trjname}_{frame}", frames=None, **kwargs):
```

Comment:
Fix the docs below that say "Only single-frame coordinate files are supported."

Reply:
Fix the docs for `file_format`.

And we have to make clearer that this can write
- coordinates (single frame *and* trajectories)
- selections

Reply:
Done.